recipe/meta.yaml

```diff
@@ -1,8 +1,8 @@
-{% set shortversion = "4.0" %}
-{% set version = "4.0.0" %}
-{% set sha256 = "8d10511df96e73b8ff9e7abbfb4d4d432edbdbe965f1f4f07afaf370b8a533be" %}
+{% set shortversion = "5.0" %}
+{% set version = "5.0.0" %}
+{% set sha256 = "e35dcbae6084adcf4abb32514127c5eabd7d63b733852ccdb31e06f1373136da" %}
 
-{% set llvm_variant = os.environ.get('LLVM_VARIANT', 'default') %}
+{% set llvm_variant = "default" %}
```

Comment:
Can you put revert this change? After the rerendering, you don't need these.

Reply:
Will do.